### PR TITLE
Feat start

### PR DESCRIPTION
**Pull request by Kothalkarpriya**
It is the first pull request after creating react app. So you can merge some files which are changed for rebuilding the website. 
Files to check:

- e-commerce/src/components/index.js
- e-commerce/src/components/navigation/Navbar.jsx
- e-commerce/src/pages/home-page/HomePage.jsx
- e-commerce/src/pages/home-page/homepage.css
- e-commerce/src/pages/index.js

The code was reviewed in the earlier repository but here are some changes made as react required.

The website looks like:
image

### CODE REVIEW

File: e-commerce/src/components/navigation/Navbar.jsx [hunk 1]
```diff
@@ -0,0 +1,34 @@
+export default function Navbar() {
+  return (
+    <nav className="navbar">
+      <div className="nav-brand upper-text">
+        <a href="/home.html" className="nav-brand-link">
```
Comment (harshalini) [inline e-commerce/src/components/navigation/Navbar.jsx:5]: You can use Link instead of "a" tag

File: e-commerce/src/pages/home-page/HomePage.jsx [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+                      src={img4}
+                      alt="sunset-and-glass-mug"
+                      className="img-horizon"
+                    />
+                  </a>
+                </div>
+                <div className="text-horizon text-align-left">
+                  <div className="text-badge">New Arrival</div>
+                  <h3>Summer Collection</h3>
+                  Check out our Best winter collection to stay warm in style
```
Comment (harshalini) [inline e-commerce/src/pages/home-page/HomePage.jsx:106]: I think this is category, you can fetch the data for category from backend

**Review by harshalini — APPROVED** [review]

**Review by JantuDeb — APPROVED** [review]
Looks good

File: e-commerce/src/components/navigation/Navbar.jsx (hunk 1 above)
Comment (Kothalkarpriya) [reply e-commerce/src/components/navigation/Navbar.jsx:5]: okay

File: e-commerce/src/pages/home-page/HomePage.jsx (hunk 2 above)
Comment (Kothalkarpriya) [reply e-commerce/src/pages/home-page/HomePage.jsx:106]: Ok I will look into it

**Pull request merged**